isacc_messaging/api/views.py

```diff
@@ -2,6 +2,7 @@
 import click
 import logging
 from datetime import datetime
+import os
```

Comment:
i think this isn't needed, now that you moved the migration code to a separate module